### DISCUSSION TITLE
0034 XLS-34d: Token-Enabled Escrows and Payment Channels

### DISCUSSION THREAD

**Discussion by dangell7** [XRPLF/XRPL-Standards#88] [XLS Proposals]
## The earlier version can be found by looking at the edit history.

**Comment by intelliot** [comment]
While we're making improvements to payment channels, consider https://github.com/XRPLF/rippled/issues/3322

**Comment by intelliot** [comment]
Proposed implementation: https://github.com/XRPLF/rippled/pull/4396

**Comment by dangell7** [comment]
Hello group: per the new [XLS Contributing process](https://github.com/XRPLF/XRPL-Standards/blob/master/CONTRIBUTING.md), it is my opinion that we have reach a "well-refined standard."

As such, I propose that we move this discussion to a file (via https://github.com/XRPLF/XRPL-Standards/pull/112) and work on final changes using additional PRs, for better change-tracking. 

Please comment here if you would like to object to moving this spec/discussion forward in the process into a DRAFT spec.

(Note that per the Contributing guidelines, moving a spec into the [DRAFT](https://github.com/XRPLF/XRPL-Standards/blob/master/CONTRIBUTING.md#drafts) state does not mean any kind of endorsement, nor does it mean that this specification will become adopted. It is solely meant as a mechanism to enable better change tracking using PRs.)

**Comment by intelliot** [comment]
Closing this discussion in order to focus future comments on https://github.com/XRPLF/XRPL-Standards/pull/112.
